Sources/Growth/URP/AttributionManager.swift

```diff
@@ -0,0 +1,203 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Preferences
+import Combine
+import Shared
+
+public enum FeatureLinkageType: CaseIterable {
+  case notdefined, vpn, playlist, leoAI
```

Comment:
Is there any gain for us for having `notdefined` value over using an optional enum?

Reply:
I thought t would be clearer but change is made https://github.com/brave/brave-ios/pull/8591/commits/032aefbf64d46f2abee234386fdc176f3a35f704